### PR TITLE
feat: add coordinates and sensor layout XML files for 10-20 layout

### PR DESCRIPTION
**Pull request by damian5710**

### CODE REVIEW

File: mffpy/resources/coordinates/Standard 10-20 (19 ch).xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<coordinates xmlns="http://www.egi.com/coordinates_mff"
```
Comment (damian5710) [inline mffpy/resources/coordinates/Standard 10-20 (19 ch).xml:2]: Should we change this to `http://www.bel.company/coordinates_mff`?

File: mffpy/resources/coordinates/Standard 10-20 (19 ch).xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                <number>131</number>
+                <type>1</type>
+                <x>0.0</x>
+                <y>0.0</y>
+                <z>0.968</z>
+            </sensor>
+        </sensors>
+    </sensorLayout>
+    <acqTime>1984-02-18T14:00:10.000000+01:00</acqTime>
+    <acqMethod>some method</acqMethod>
```
Comment (damian5710) [inline mffpy/resources/coordinates/Standard 10-20 (19 ch).xml:162]: Should we use other values here?

**Review by marianofco — CHANGES_REQUESTED** [review]
There is something wrong with some electrodes, at least with the Pz, P3, P4, P7, P8, and probably O1 and O2:
image

File: mffpy/resources/coordinates/Standard 10-20 (19 ch).xml (hunk 2 above)
Comment (jusjusjus) [reply mffpy/resources/coordinates/Standard 10-20 (19 ch).xml:186]: Not sure but these fields might not be needed at all.  You'll need to check the EGI software for compatibility.

Comment (marianofco) [reply mffpy/resources/coordinates/Standard 10-20 (19 ch).xml:186]: I would say use an actual date-time
And for the acquisition method, and following other xml files, I would say "Atlas based coordinates" to give a hint about where we took these coordinates from.

**Review by marianofco — CHANGES_REQUESTED** [review]
Coordinates are not symmetric, compared to for instance GSN256, but it shoulddn't be an issue right now:

image

(left: GSN 256 and right GHW 130)

- Please modify the date time and acq mode as commented

File: mffpy/resources/coordinates/Standard 10-20 (19 ch).xml (hunk 2 above)
Comment (damian5710) [reply mffpy/resources/coordinates/Standard 10-20 (19 ch).xml]: Done!

**Review by marianofco — APPROVED** [review]

**Pull request merged**